src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration.mdx

```diff
@@ -0,0 +1,165 @@
+  - AWS integrations list
+metaDescription: "New Relic's AWS Lex monitoring integration: what data it reports, and how to enable it."
+redirects:
+  - /docs/integrations/amazon-integrations/aws-integrations-list/aws-lex-monitoring-integration
+  - /docs/aws-lex-integration
+  - /docs/infrastructure/infrastructure-integrations/amazon-integrations/aws-lex-integration
+  - /docs/infrastructure/amazon-integrations/amazon-integrations/aws-lex-monitoring-integration
+---
+
+[New Relic infrastructure integrations](/docs/infrastructure/integrations-getting-started/getting-started/introduction-infrastructure-integrations) include an Amazon Lex integration for reporting your Lex data to New Relic. Here we explain how to activate our integration and what data can be reported.
+
+Enable the [AWS CloudWatch Metric Streams integration](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-metric-stream/) to monitor all CloudWatch metrics from your AWS services, including custom namespaces. Individual integrations are no longer our recommended option.
+
+## Features [#features]
+
+[Amazon Lex](https://aws.amazon.com/lex/) (Lex) is an AWS service for building conversational interfaces into applications using voice and text. With New Relic's Lex integration, you'll be able to monitor number of requests, channels used for Bot conversation and more. You'll be able to create custom charts of your data and [create alert conditions](/docs/infrastructure/new-relic-infrastructure/infrastructure-alert-conditions/infrastructure-alerts-add-edit-or-view-host-alert-information) based on changes in Lex data or configuration.
+
+## Activate integration [#activate]
+
+To enable this integration follow standard procedures to [connect AWS services to New Relic](/docs/infrastructure/infrastructure-integrations/getting-started/connect-aws-integrations-infrastructure).
+
+## Configuration and polling [#polling]
+
+You can change the polling frequency and filter data using [configuration options](/docs/integrations/new-relic-integrations/getting-started/configure-polling-frequency-data-collection-cloud-integrations).
+
+These are the default [polling](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-polling-intervals-infrastructure-integrations) values for the Amazon EBS integration:
+
+* New Relic polling interval: 15 minutes
+* Amazon CloudWatch data interval: [1 minute or 5 minutes](http://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/ebs-metricscollected.html), depending on volume type
+
+## Find and use data [#find-data]
+
+To find this integration's data, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > AWS** and select one of the Amazon Lex integration links.
+
+You can [query and explore your data](/docs/using-new-relic/data/understand-data/query-new-relic-data) using the `LexSample` [event type](/docs/data-apis/understand-data/new-relic-data-types/#event-data), with a `provider` value of `Lex`.
```

Comment:
Is there really a LexSample event table? I thought everything ends up in Metric now?

Reply:
All other documentation is also maintained like this. This LexSample is defined in infra-metrics file

Reply:
addressed